src/Codebase/Infrastructure/SourceParser/SourceParser.php

```diff
@@ -63,6 +64,9 @@ public function parseSource(SourceParserRequestDto $codebaseRequestDto): Codebas
             }
             $codebaseSourceDto = $codebaseSourceDto->setType($type);
```

Comment:
Do we need this property as we just analyze project codebase?

Reply:
We still need to analyze the vendor for Transfer and BD entity checks (only .XML files)